app/views/schools/cohorts/add_participants.html.erb

```diff
@@ -1,24 +1,22 @@
+<% content_for :title, "Add teachers and mentors" %>
+
 <% content_for :before_content, govuk_back_link(
   text: "Back",
-  href: schools_cohort_path("2021"))
+  href: schools_cohort_path(@cohort.start_year))
 %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-m">2021</span>
-    <h1 class="govuk-heading-l">Add teachers and mentors</h1>
-
-    <p class="govuk-body">From <strong>July</strong>, you can confirm which early career teachers and mentors are starting a programme this year.</p>
-    <p class="govuk-body">We will let you know when you can get started.</p>
-
-    <p class="govuk-body">For each person, you will need:</p>
-
+    <span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
```

Comment:
Let's use `@cohort.display_name` here. They're the same at the moment, but they have changed before